pkgs/development/dotnet-modules/python-language-server/deps.nix

```diff
@@ -1,1231 +1 @@
-# This file is autogenerated.
-# To regenerate, run "create_deps.sh $PATH_TO_LANGUAGE_SERVER_CHECKOUT"
 
```

Comment:
This file should not be empty.

Reply:
not sure what happened here. should be fixed.